
## SGE_BLOC2

#### 090125

Comprobando la conexión con este archivo, he agregado un print, para confirmarlo.

```
![Captura de pantalla psycopg0 connect.py] (home/isard/Imágenes/Capturas de pantalla/20250109.png)
```

#### 160125

Insercion del registro del formato csv a la Base de Datos.

```
![Captura de pantalla psycopg0 connect.py] (home/isard/Imágenes/Capturas de pantalla/20250216.png)
```

#### 230125

Insercion del nuevo registro(modificado por mi, despues de haber solucionado los errores) con el script a la Base de Datos.

```
![Captura de pantalla psycopg0 connect.py] (home/isard/Imágenes/Capturas de pantalla/20250123.png)
```

Resultado de ' print(results) ' en read_registre.py

```
![Captura de pantalla psycopg0 connect.py] (home/isard/Imágenes/Capturas de pantalla/20250230.png)
```

Y tambien del print a la fila 5

```
![Captura de pantalla psycopg0 main.py] (home/isard/Imágenes/Capturas de pantalla/20250206.png)
```

Ahora el teléfono

```
![Captura de pantalla psycopg0 main.py] (home/isard/Imágenes/Capturas de pantalla/0206.1.png)
```

Correcion del código..
```
print(results[4][2])
```

#### 



Alvaro H. DAM 1B
